Training/Training/TestCase.c

```diff
@@ -0,0 +1,78 @@
+// ------------------------------------------------------------------------------------------------
+// Assignment3 ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// DecimalConversion.c
+// C program to check the test cases of Decimal conversion
+// ------------------------------------------------------------------------------------------------
+
+#include <stdio.h>
+#include <string.h>
+#include "DecimalConversion.h"
+
+void main () {
+   //test cases for Binary conversion
+   printf ("-----Decimal to Binary-----\n");
+   int input[] = { 0,651,-545,999,-4875,15756,-4654,-1,-2147483647 };
+   char* output1[] = {
+      "00000000000000000000000000000000",
+      "00000000000000000000001010001011",
+      "11111111111111111111110111011111",
+      "00000000000000000000001111100111",
+      "11111111111111111110110011110101",
+      "00000000000000000011110110001100",
+      "11111111111111111110110111010010",
+      "11111111111111111111111111111111",
+      "10000000000000000000000000000001" };
+   for (int i = 0; i < 9; i++) {
+      int num1 = input[i];
+      printf ("Test Case%d: Input--> %d ", i + 1, num1);
+      char* binResult1 = DecToBin (num1);
+      int m = strcmp (binResult1, output1[i]);
+      if (m == 0) printf ("\033[1;32mPASS\033[0m\n");
+      else printf ("\033[1;31mFAIL\033[0m\n");

```

Comment:
instead of  passing color codes every time  better define it printf ("\033[1;32mPASS\033[0m\n")
![image](https://github.com/user-attachments/assets/9d491630-dc08-4557-adb1-cf1d594898c5)
this makes your code more readable